.github/workflows/gradle.yml

```diff
@@ -0,0 +1,17 @@
+name: Java CI
+
+on: [push]
```

Comment:
Does this mean this action be triggered whenever we push to master? If so, is it too late? I can see Alerting plugin's action is on Pull Request.

Reply:
Done

Reply:
Thanks!